doc/conseil.sql

```diff
@@ -2,10 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 9.5.17
--- Dumped by pg_dump version 11.4 (Ubuntu 11.4-1.pgdg18.04+1)
-
--- Started on 2019-06-27 13:32:57 EDT
+-- Dumped from database version 11.4
```

Comment:
We run Postgres 9 in production. Could you dump this file for that version so we don't have compatibility issues?

Reply:
I deleted the only difference between this version and postgres 9 as a temp fix. I'll need to uninstall/reinstall Postgres for the future.